openrtb_ext/imp_rise.go

```diff
@@ -5,4 +5,5 @@ type ImpExtRise struct {
 	PublisherID string `json:"publisher_id"`
 	Org         string `json:"org"`
 	PlacementID string `json:"placementId"`
+	TestMode    bool   `json:"testMode"`
```

Comment:
@SamuelRise : Being its impression level parameter, curious to understand how `testEndpointURL` and `endpointURL` will work in following scenario - 

```json
{
    "imp": [
        {
            "id": "imp_1",
            "ext": {
                "prebid": {
                    "bidder": {
                        "rise": {
                            "testMode": true
                        }
                    }
                }
            }
        },
        {
            "id": "imp_2",
            "ext": {
                "prebid": {
                    "bidder": {
                        "rise": {
                            "testMode": false
                        }
                    }
                }
            }
        }
    ]
}
```